src/reducers/alerts.js

```diff
@@ -19,6 +19,7 @@ const reducer = function (state, action) {
             const extension = extensionData.find(ext => ext.extensionId === extensionId);
             if (extension && extension.name) {
                 // TODO: is this the right place to assemble this message?
+                newAlert.extensionId = extensionId;
```

Comment:
Maybe the data about if an alert should show the reconnect button should be set explicitly here, so that we can decide whether to display it or not. It looks like we currently want to display it for every alert associated with an extension, so for now maybe we should set `newAlert.showReconnect` if `extensionId` is truthy?  Then we can pass that as a prop to the alert component.

Reply:
Yes, I believe this is now done in #3559

Reply:
I'll pick this up in #3559